docs/_includes/common/pagination.html

```diff
@@ -49,7 +49,7 @@ <h2 class="clipped" id="pagination-heading">Results Pagination - Page 1</h2>
             <a aria-current="page" href="http://www.ebay.com/sch/i.html?_nkw=guitars" class="pagination__item">1</a>
         </li>
         <li>
-            <a href="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=2" class="pagination__item">2</a>
+            <button data-action="http://www.ebay.com/sch/i.html?_nkw=guitars&_pgn=2" class="pagination__item">2</a>
```

Comment:
Were these changes to simply test the new selector? We'd likely not be mixing `<a>` and `<button>` in the same pagination component, would we?

Reply:
Maybe we should just add another example that uses only buttons.

Reply:
This was done just to illustrate that you can use a button also. I don't know that a whole new example is needed.

Reply:
Will our developers be able to see this, though? It seems kinda buried.

Reply:
Yeah, I'd hate to set the example that they should mix them.  I agree that another example would be better.